### ISSUE TITLE
Start from empty

### ISSUE THREAD

**Issue by koba-1** [BuddyLReno/smarquee#7]
Is it possible to start the animation with the empy field? That is, at the beginning it is empty and the text appears from outside the margins immediately after...

**Comment by BuddyLReno** [comment]
No, it uses the starting text to determine how long it needs to scroll.
